src/topology/category/TopCommRing.lean

```diff
@@ -35,8 +38,12 @@ instance : concrete_category TopCommRing.{u} :=
         dsimp, apply continuous.comp ; assumption
       end⟩ },
   forget := { obj := λ R, R, map := λ R S f, f.val },
-  forget_faithful := { }
-}
+  forget_faithful := { } }
+end
+
+instance (X : TopCommRing) : comm_ring X := X.is_comm_ring
```

Comment:
```suggestion
instance (R : TopCommRing) : comm_ring R := R.is_comm_ring
```
Sorry for all these annoying suggestions... but I think eye candy is important PR. And eye-candy is one of the things where the category lib high potential.
(Of course the fact that we have to write these instances is bad... but I think it is non-trivial to automate them away.